python/cugraph/cugraph/tests/sampling/test_random_walks.py

```diff
@@ -70,25 +71,30 @@ def calc_random_walks(graph_file, directed=False, max_depth=None, use_padding=Fa
     sizes: int
         The path size in case of coalesced paths.
     """
-    G = graph_file.get_graph(create_using=cugraph.Graph(directed=directed))
     assert G is not None
 
-    k = random.randint(1, 10)
+    k = random.randint(1, 6)
+
     random_walks_type = "uniform"
-    start_vertices = random.sample(range(G.number_of_vertices()), k)
+
+    start_vertices = G.select_random_vertices(num_vertices=k)
```

Comment:
According to the docstring, `G` can be a Nx Graph, but I don't think that has a `select_random_vertices()` method.